app/ModelFunctions/SymLinkFunctions.php

```diff
@@ -64,4 +88,21 @@ public function clearSymLink()
 
 		return $no_error ? 'true' : 'false';
 	}
+
+	/**
+	 * Remove outdated SymLinks.
+	 *
+	 * @return bool
+	 */
+	public function remove_outdated()
+	{
+		$symlinks = SymLink::where('created_at', '<', now()->subDays(intval(Configs::get_value('SL_life_time_days', '7')))->toDateTimeString())->get();
```

Comment:
Should the fallback here be `7` or `1`?

Reply:
I don't know how long we want our symlink to stay live by default. 1 day ? 3 days ? 7 days ?

Reply:
Sorry, I should've been more clear what I meant. Right now the fallback (`7`) is inconsistent with the default value set in the database migration (`1`). I believe they should be the same. This is what I meant with my question.

Reply:
Yeah I know. But I'm still undecided on the default value. ^^

Reply:
let's go for 3.